### PR TITLE
Branch 1 commit

### PR DESCRIPTION
**Pull request by vidovgopol**

### CODE REVIEW

**Comment by vidovgopol** [comment]
Commit this.

**Review by SanchezRick — APPROVED** [review]
I approve this change.

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (voronov007) [inline .idea/misc.xml:1]: add `.idea` and `.idea/` to gitiugnore. Delete these unnecessary files

**Review by voronov007 — APPROVED** [review]
Looks good

**Pull request merged**